common-content/en/module/tools/converting-javascript-to-python/index.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "Converting JavaScript to Python"
+headless = true
+time = 40
+facilitation = false
+emoji= "📖"
+objectives = [
+    "Rewrite JavaScript code as Python"
+]
++++
+
+### Parsing command line flags
+
+In JavaScript, we wrote this code (note: there was some other code in between some of these lines):
+
+```js
+import { program } from "commander";
+
+program
+    .name("count-containing-words")
+    .description("Counts words in a file that contain a particular character")
+    .option("-c, --char <char>", "The character to search for", "-");
+
+program.parse();
+
+const argv = program.args;
+const path = argv[0];
+const char = program.opts().char;
+```
+
+The _essential_ goals here are to:
```

Comment:
Again, make this an activity -- get them to identify the essential goals. 

Sorting and categorising activities are a key way humans build up their mental model, so let them get some foundations laid here. You could provide this in a few ways here

1  - give them all the goals and have them sort them into essential and accidental
2- have them identify the goals themselves and provide a key

Or another option I haven't considered...